Dockerfile

```diff
@@ -62,13 +62,16 @@ RUN mkdir /opt/taudem && \
 ENV PATH /opt/taudem/:$PATH
 
 # Download and build RWD
+# The SHA in the following URL should be kept in sync with the latest commit on
+# the develop branch. If we simply used develop.tar.gz, this command would
+# not be executed (due to the cache) and we would not get the latest commits.
+# See https://ryanfb.github.io/etc/2015/07/29/git_strategies_for_docker.html#dockerfile-inside-git-repository
 RUN mkdir /opt/rwd/ && \
-    wget -qO- https://github.com/WikiWatershed/rapid-watershed-delineation/archive/develop.tar.gz \
+    wget -qO- https://github.com/WikiWatershed/rapid-watershed-delineation/archive/edcf497867355d039f721ecaa6ff663fde720f3b.tar.gz \
```

Comment:
Does the `archive` endpoint work with release tags?  Would be great to do a github release and update this url for using the new code (and using commit sha's for development)